content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -427,7 +427,44 @@ For a TCP probe, the kubelet makes the probe connection at the node, not in the
 means that you can not use a service name in the `host` parameter since the kubelet is unable
 to resolve it.
 
+### Setting TerminationGracePeriodOverride
 
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+Prior to release 1.21, the pod-level `terminationGracePeriodSeconds` was used
+for terminating a container that failed its liveness or startup probe.
+
+In 1.21, when the feature flag `ProbeTerminationGracePeriod` is enabled, a user
+can specify a probe-level `terminationGracePeriodSeconds` as part of the probe
+specification. When the feature flag is enabled, and both a pod- and
+probe-level `terminationGracePeriodSeconds` are set, the kubelet will use the
+probe-level value.
+
+For example,
+
+```yaml
+spec:
+  terminationGracePeriodSeconds: 3600
+  containers:
+  - name: test
+    image: ...
+
+    ports:
+    - name: liveness-port
+      containerPort: 8080
+      hostPort: 8080
+
+    livenessProbe:
+      httpGet:
+        path: /healthz
+        port: liveness-port
+      failureThreshold: 1
+      periodSeconds: 60
+      # Override pod-level terminationGracePeriodSeconds #
+      terminationGracePeriodSeconds: 60
+```
+
+Probe-level `terminationGracePeriodSeconds` cannot be set for readiness probes.
```

Comment:
nit - can we please be clear if this will this result in failure? or be silently ignored. 

Looks great! thanks @ehashman

Reply:
It's validated by the API server so it'll be rejected outright. Not sure how to phrase that here...

Reply:
Any wording that's technically correct is OK for now - we can tidy that kind of detail after the merge.

Reply:
I'm all for clarity though!